run/ktor-hello-world/CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```

Comment:
We can remove this file because we already have a [CONTRIBUTING.md](https://github.com/GoogleCloudPlatform/kotlin-samples/blob/master/CONTRIBUTING.md) in the repo root which covers this sample